components/formats-bsd/src/loci/formats/in/FilePatternReader.java

```diff
@@ -133,7 +133,11 @@ public FilePatternReader() {
     int fileIndex = fileIndexes[getSeries()][no][0];
     int planeIndex = fileIndexes[getSeries()][no][1];
     helper.setId(files[fileIndex]);
-    return helper.openBytes(planeIndex, buf, x, y, w, h);
+    int oldSeries = helper.getSeries();
+    helper.setSeries(getSeries());
```

Comment:
This will break the case where ```FilePatternReader``` has multiple series, but each file has a single series.  I don't know off hand if any of the IDR datasets exhibit this, but would be worth verifying before merging.

I'd also expect just ```helper.setSeries(getSeries())``` to be sufficient (without saving/resetting the series afterwards), but if there is a reason for the extra calls it might be good to document it.